src/components/lib/styles/__style-guide.css

```diff
@@ -4,9 +4,9 @@
   --primary-dark: #141414;
   --text-white: #fff;
   --text-grey: #8d8d8d;
-  --colors-green: #8fff00;
-  --colors-yellow: #ffc226;
-  --colors-orange: #ff6826;
+  --color-green: #8fff00;
```

Comment:
I feel the green is too contrast, can we use bit mild

Reply:
Updated the screenshot: https://user-images.githubusercontent.com/24249988/173537999-6772c6e9-ee89-42df-a4aa-32591b258737.png